Examples/graph-toy.lisp

```diff
@@ -99,21 +99,22 @@
   ((val :initarg :val :accessor val)))
 
 (defmethod handle-event ((frame graph-toy) (event new-value-event))
-  (vector-push-extend (val event) (val-array frame))
-  (let ((val-array-len (length (val-array frame)))
-        (xval-count (max-xvals frame)))
-    (when (>= val-array-len (* xval-count 2))
-      (loop for i from 0 to (- xval-count 1)
-            for j from (- val-array-len xval-count)
-            do (setf (elt (val-array frame) i) (elt (val-array frame) j)))
-      (setf (fill-pointer (val-array frame)) xval-count))
-    (redisplay-frame-pane frame 'main-display)
-    (incf (last-yval frame) (- (random 9) 4))
-    (when (member (frame-state frame) '(:enabled :shrunk))
-      (schedule-event (frame-top-level-sheet frame)
-                      (make-instance 'new-value-event :sheet frame
-                                                      :val (last-yval frame))
-                      0.5))))
+  (when (eq (frame-state frame) :enabled)
```

Comment:
`frame-state` could return 4 possible values, `:disowned`, `:disabled`, `:enabled` or `:shrunk`. We definitely do not want to handle events while `:disowned` or `:disabled`. You are sure about handling events when `:enabled`, then what do you think should happen when the frame state is `:shrunk`?